docker/neurodocker_gen_base.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+docker run --rm repronim/neurodocker:latest \
```

Comment:
The dockerfile is completely autogenerated, minimal maintenance.
Couldn't get SPM to work in a generated image, may need manual maintenance. But then it's not part of our workflow yet. Get back to it when we actually need it.